### ISSUE TITLE
Class-based response codes are not being used when written in XML comment

### ISSUE THREAD

**Issue by JustArchi** [domaindrivendev/Swashbuckle.AspNetCore#900]
Hello, first of all thank you for awesome library.

I'd like to report a bug in which class-based XML comments are not being generated in swagger specification. Example class to reproduce the problem:

```csharp
/// <summary>
/// This is class description, not relevant.
/// </summary>
/// <response code="400">This should show.</response>
/// <response code="401">This too!</response>
[ApiController]
[Route("Api/MyController")]
public sealed class MyController : ControllerBase {
	/// <summary>
	/// Method description.
	/// </summary>
	[HttpGet]
	[ProducesResponseType(typeof(string), 200)]
	public ActionResult<string> Get() {
		return Ok("");
	}
}
```

Using code above results only in `200` code being shown as possible response, due to local `ProducesResponseType` attribute that is applied directly to method. Adding (redundant) `ProducesResponseType` attributes to class will cause them to be displayed but **without** description (which can't be put in `ProducesResponseType` in the first place). XML `response` tags seem to be entirely ignored as they were never there.

However, using `SwaggerResponse` attribute instead of response codes directly in XML causes them to display properly:

```csharp
/// <summary>
/// This is class description, not relevant.
/// </summary>
[ApiController]
[Route("Api/MyController")]
[SwaggerResponse(400, "This shows now.")]
[SwaggerResponse(401, "And this too!")]
public sealed class MyController : ControllerBase {
	/// <summary>
	/// Method description.
	/// </summary>
	[HttpGet]
	[ProducesResponseType(typeof(string), 200)]
	public ActionResult<string> Get() {
		return Ok("");
	}
}
```

The above code achieves my wanted scenario where `Get()` method is shown with 3 possible response codes - 400, 401 and 200.

The issue is reproducible only with class-wide response codes, XML comments work OK within local methods, so there is no mistake on my part related to XML comments generation. I don't see any reason why my XML comment would work fine in a method but not a class, considering `SwaggerResponse` works in both, it's inconsistency at best, but likely a bug.

Thank you in advance for taking a look into my case. I strongly believe that this is a bug, but let me know in case I'm wrong. I'm trying to declare some response codes as class-wide for entire controller instead of duplicating them in each method. This is possible by decorating my class with `SwaggerResponse`, but apparently it's not possible doing the same with XML comments, contrary to local methods that respect both ways.

This was reproduced with latest `Swashbuckle.AspNetCore` as available on nuget, version `3.0.0`.

**Comment by liguobao** [comment]
add code 
```C#
services.AddSwaggerGen(c =>
{
    c.SwaggerDoc("v1",
        new Info
        {
            Title = "My API - V1",
            Version = "v1"
        }
     );

     var filePath = Path.Combine(System.AppContext.BaseDirectory, "MyApi.xml");
     c.IncludeXmlComments(filePath);
}
```

to Startup.cs ?

**Comment by JustArchi** [comment]
@liguobao 

> The issue is reproducible only with class-wide response codes, XML comments work OK within local methods, so there is no mistake on my part related to XML comments generation. 

I'm already using similar code.

**Comment by domaindrivendev** [comment]
Legitimate issue but I'm downgrading (i.e. putting it on the Backlog) so it doesn't hold up the 5.0.0 release.

**Comment by domaindrivendev** [comment]
Not sure exacly when this feature was added but it's certainly available in `v5.6.0`
